app/controllers/admin/pages_controller.rb

```diff
@@ -1,67 +1,67 @@
 module Admin
   class PagesController < ApplicationController
-    
+
     before_action :authenticate_user
     respond_to :html
-    
+
     def index
       @pages = Page.roots
     end
-    
+
     def new
       @page = Page.new
     end
-    
+
     def create
       @page = Page.new(page_params)
-  
+
       if @page.save
         redirect_to admin_pages_path, notice: 'Page was successfully created.'
       else
         render action: "new"
       end
     end
-    
+
     def edit
       @page = Page.find_by(slug: params[:id])
     end
-    
+
     def update
       @page = Page.find_by(slug: params[:id])
-  
+
       if @page.update_attributes(page_params)
         redirect_to admin_pages_path, notice: 'Page was successfully updated.'
       else
         render action: "edit"
       end
     end
-    
+
     def destroy
       @page = Page.find_by(slug: params[:id])
       @page.destroy
-      
-      redirect_to admin_pages_path, :notice => 'Page was successfully deleted.'
+
+      redirect_to admin_pages_path, notice: 'Page was successfully deleted.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.